vocabularies/Common.xml

```diff
@@ -371,22 +371,33 @@
+          <String>If this annotation is given dynamically in the metadata, its value in the before-image and after-image of a request
+          can play a role when changes to a property are requested.</String>
+        </Annotation>
         <Member Name="Mandatory" Value="7">
           <Annotation Term="Core.Description" String="Property is mandatory from a business perspective" />
-          <Annotation Term="Core.LongDescription" String="This annotation value does not imply any restrictions on the value range of the property. For restricting the value range use e.g. the standard type facet `Nullable` with a value of `false` to exclude the `null` value, or terms from the [Validation vocabulary](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Validation.V1.md)." />
+          <Annotation Term="Core.LongDescription">
+            <String>A requested removal of the property is rejected if this annotation is `Mandatory` in the after-image of the request.
+            This annotation value does not imply any restrictions on the value range of the property. For restricting the value range use e.g. the standard type facet `Nullable` with a value of `false` to exclude the `null` value, or terms from the [Validation vocabulary](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Validation.V1.md).</String>
+          </Annotation>
         </Member>
         <Member Name="Optional" Value="3">
           <Annotation Term="Core.Description" String="Property may have a value" />
           <Annotation Term="Core.LongDescription" String="This value does not make sense as a static annotation value." />
         </Member>
         <Member Name="ReadOnly" Value="1">
           <Annotation Term="Core.Description" String="Property value cannot be changed" />
-          <Annotation Term="Core.LongDescription" String="To statically mark a property as read-only use term [Core.Computed](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed) instead" />
+          <Annotation Term="Core.LongDescription">
+            <String>A requested change of the property is rejected if this annotation is `ReadOnly` in the before-image _and_ the after-image of the request.
+            To statically mark a property as read-only use term [Core.Computed](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed) instead.</String>
+          </Annotation>
         </Member>
         <Member Name="Inapplicable" Value="0">
           <Annotation Term="Core.Description" String="Property has no meaning in the current entity state" />
           <Annotation Term="Core.LongDescription">
-            <String>This value does not make sense as a static annotation value.
+            <String>A requested change mentioning the property is rejected if this annotation is `Inapplicable` in the after-image of the request.
```

Comment:
What if the property value is set to `null` at the same time as setting the property to `Inapplicable`? That seems to make sense.